pkg/rbac/cache.go

```diff
@@ -0,0 +1,95 @@
+package rbac
+
+import "time"
+
+//to get the token and uid
+func (tr *UserTokenReviews) GetTimebyToken(token string) (interface{}, bool) {
+	if THREADING {
+		tr.lock.RLock()
+		defer tr.lock.RUnlock()
+
+	}
+	value, exists := tr.data[token]
+	if !exists {
+		return value, false
+	}
+	return value, true
```

Comment:
This would do the same.
```suggestion
	return value, exists
```